### DISCUSSION TITLE
Welcome to CityCatalyst Discussions!

### DISCUSSION THREAD

**Discussion by evanp** [Open-Earth-Foundation/CityCatalyst#77] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by praveensinghrajput23** [comment]
Hello everyone,

I'm Praveen Singh, a passionate software developer and dedicated open-source contributor. I'm thrilled to be part of this global community of brilliant minds united by a common goal: making a positive impact on our beloved planet.

@evanp, let's collaborate and come up with a motto that embodies our shared mission. Together, we can innovate and contribute to a better future for Earth.

Thank you for the opportunity to join this inspiring journey.
